pkg/arvo/sys/vane/ames.hoon

```diff
@@ -1756,7 +1756,10 @@
         14+(state-13-to-14:load:adult-core +.u.cached-state)
       =?  u.cached-state  ?=(%14 -.u.cached-state)
         15+(state-14-to-15:load:adult-core +.u.cached-state)
-      =?  u.cached-state  ?=(%15 -.u.cached-state)
+      =^  moz  u.cached-state
+        ?.  ?=(%15 -.u.cached-state)  [~ u.cached-state]
+        ~>  %slog.0^leaf/"ames: init dead flow consolidation timer"
+        :-  [[/ames]~ %pass /dead-flow %b %wait `@da`(add now ~m2)]~
```

Comment:
Do we need to duplicate this timer initialization somewhere else to catch fresh boot?